src/test/strategy.test.ts

```diff
@@ -0,0 +1,40 @@
+import { SupabaseStrategy } from '../index'
+import { supabaseClient } from '../mocks/supabase'
+import { sessionStorage } from '../mocks/sessionStorage'
+
+describe('strategy', async() => {
+  it('should throw if missing supabaseClient', async() => {
+    expect.assertions(1)
```

Comment:
expect assertions is usually not really needed. You mostly want to add it whenever you do async stuff that could possibly never the reach the assertions.